src/com/superiornetworks/icarus/commands/Command_smite.java

```diff
@@ -48,8 +52,8 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
         player.setHealth(0D);
         player.setOp(false);
         player.setGameMode(GameMode.SURVIVAL);
-        Bukkit.broadcastMessage(ChatColor.RED + player.getName() + " has been very naughty, indeed.");
-        player.sendMessage(ChatColor.RED + "You were smitten by: " + sender.getName());
+        Bukkit.broadcastMessage(ChatColor.RED + player.getName() + " has been very naughty, indeed.\nThey have thus been smitten.\nReason - " + ChatColor.YELLOW + smiteReason);
```

Comment:
I personally do not like "They have thus been smitten"

Maybe just keep the reason part.